src/makeInfinityAdder.js

```diff
@@ -4,7 +4,21 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  return function makeAdder(arg) {
```

Comment:
The inner function `makeAdder` should be named differently or be an anonymous function since its name does not add clarity and can be confused with the outer function's purpose.